### PR TITLE
Update version to 0.7.3, update HISTORY.md, query layout fix for dense path

### PR DESCRIPTION
**Pull request by ihnorton**

### CODE REVIEW

**Review by Shelnutt2 — APPROVED** [review]

File: tiledb/multirange_indexing.py [hunk 1]
```diff
@@ -128,14 +128,23 @@ def __getitem__(self, idx):
...
 
-        # default TILEDB_UNORDERED for sparse, TILEDB_ROW_MAJOR for dense
-        order = 'U' if self.schema.sparse else 'C'
+        # set default order
+        # - TILEDB_UNORDERED for sparse
+        # - TILEDB_ROW_MAJOR for dense
+        if self.schema.sparse:
+            order = 'U'
+        else:
+            order = 'C'
```
Comment (Shelnutt2) [inline tiledb/multirange_indexing.py:137]: Isn't this column major? The comment says row?

File: tiledb/libtiledb.pyx [hunk 2]
```diff
@@ -4227,14 +4227,22 @@ cdef class Query(object):
...
         self.attrs = attrs
         self.coords = coords
-        self.order = order
         self.return_arrow = return_arrow
+
+        if order == None:
+            if array.schema.sparse:
+                self.order = 'U'
+            else:
+                self.order = 'C'
```
Comment (Shelnutt2) [inline tiledb/libtiledb.pyx:4242]: Is this suppose to be column or row major ordering?

File: tiledb/multirange_indexing.py (hunk 1 above)
Comment (ihnorton) [reply tiledb/multirange_indexing.py:137]: This is numpy naming semantics, `C` is "C ordering" == "row major", `F` is "fortran ordering" == "row major".

File: tiledb/libtiledb.pyx (hunk 2 above)
Comment (ihnorton) [reply tiledb/libtiledb.pyx:4242]: Same as above

**Pull request merged**